src/einsteinpy/hypersurface/schwarzschildembedding.py

```diff
@@ -174,34 +172,3 @@ def get_values_surface(self, alpha):
 
         Z.reshape((X.shape[0], X.shape[1]))
         return X, Y, Z
-
-    def plot_hypersurface(self, plot_type="wireframe", alpha=100):
```

Comment:
Preserve the function and change the code internally to get results from the plotting module.
We don't want to break APIs. We can put a `DeprecationWarning`

Reply:
This new 0.3 is all about API breaks!

Reply:
Now that we have decided to release `0.2.2`, please take care of this. 🙏

Reply:
Alright. Without my heart to it, but I will take care of that!